arviz/plots/backends/matplotlib/pairplot.py

```diff
@@ -246,11 +245,29 @@ def plot_pair(
         point_estimate_marker_kwargs.setdefault("s", markersize + 50)
 
         if ax is None:
+            if backend_kwargs.pop("sharex", None) is not None:
+                warnings.warn(
+                    "'sharex' keyword is ignored. For non-standard sharing, provide 'ax'.",
+                    UserWarning,
+                )
+            if backend_kwargs.pop("sharey", None) is not None:
+                warnings.warn(
+                    "'sharey' keyword is ignored. For non-standard sharing, provide 'ax'.",
+                    UserWarning,
+                )
+            backend_kwargs["sharex"] = "col"
+            if not_marginals:
+                backend_kwargs.setdefault("sharey", "row")
```

Comment:
```suggestion
                backend_kwargs["sharey"] = "row"
```

It makes no difference because the key if present has been removed, but I'd use the same method to set both sharex and sharey.

Reply:
Whoops, I actually meant to do that, just forgot to change that line!